pkg/ingestor/parser/scorecard/parser_scorecard.go

```diff
@@ -20,25 +20,25 @@ import (
 	"encoding/json"
 	"fmt"
 	"strings"
+	"time"
 
 	"github.com/guacsec/guac/pkg/assembler"
+	"github.com/guacsec/guac/pkg/assembler/graphql/model"
```

Comment:
Will need to change import to be the same as in #522, but the structs would stay similar (just in different package)